src/args.rs

```diff
@@ -108,4 +113,27 @@ impl Args {
             .column(self.column())
             .build(buffer_writer.buffer())
     }
+
+    pub(crate) fn language(&self) -> Option<SupportedLanguage> {
```

Comment:
Went further in the "methods on `Args`" direction, all of these effectively just got moved here

Reply:
Could make the `language` field private then? (Maybe? It doesn't seem like a crazy thing to have public, but it's not used anywhere else.)

Reply:
Actually, you could probably have clap directly support `SupportedLanguage` by implementing `FromStr` (or whatever the trait was) on it?

Reply:
Ah yeah, I didn’t think about the help text. I don't see an obvious way to do that in clap without an enum.

It does look like it could be an enum fairly easily. I guess there are some (probably not important) performance things to consider there since it'd be less easy to make sure the per language code only really runs once? (Which is handled in the setup of the SupportedLanguage now?)